shinydashboard/lantern/server.R

```diff
@@ -12,64 +12,80 @@ function(input, output, session) { #nolint
     }
   }, priority = 100)
 
+  observeEvent(database_fetch, {
+    if (database_fetch() == 1) {
+      show_modal_spinner(
+        spin = "double-bounce",
+        color = "#112446",
+        text = "Please Wait, Loading...")
```

Comment:
Can we change this to "Please Wait, Lantern is fetching the most up-to-date data"